README.md

```diff
@@ -442,6 +442,7 @@ Services to securely store your Docker images.
 * [Eclipse Che](http://www.eclipse.org/che/) - Developer workspace server with Docker runtimes, cloud IDE, next-generation Eclipse IDE
 * [forward2docker](https://github.com/bsideup/forward2docker) - Utility to auto forward a port from localhost into ports on Docker containers running in a boot2docker VM by [@bsideup](https://github.com/bsideup)
 * [GoSu](https://github.com/tianon/gosu) - Run this specific application as this specific user and get out of the pipeline (entrypoint script tool) by [@tianon](https://github.com/tianon)
+* [netshoot](https://github.com/nicolaka/netshoot) - The netshoot continaer has a powerful set of networking tools to help troubleshoot Docker networking issues by [@nicolaka](https://github.com/nicolaka)
```

Comment:
Hi @veggiemonk!! I was torn between it being a utility or a networking tool. The only reason I didn't put it under networking as everything else in the section is an actual network plugin. But I think you are right it would fit better in networking. Moving now.